frontend/libs/studio-components/src/components/StudioTextResourceInput/StudioTextResourceInput.module.css

```
@@ -0,0 +1,34 @@
+.container {
+  display: grid;
+  gap: var(--fds-spacing-1);
+  grid-template-areas:
+    'inputbox toggle'
+    'id toggle';
+  grid-template-columns: 1fr auto;
+}
+
+.inputbox {
+  grid-area: inputbox;
+}
+
+.id {
+  align-items: center;
+  display: flex;
+  gap: var(--fds-spacing-1);
+  grid-area: id;
+}
+
+.id code {
+  align-items: center;
+  align-self: stretch;
+  display: flex;
+}
+
+.toggle {
+  grid-area: toggle;
+}
+
+.toggle [role='radiogroup'] {
+  box-sizing: border-box;
+  height: 100%;
+}
```

Endringer gjort for å få samme resultat som vist på bilde
```suggestion
.container {
  display: grid;
  gap: var(--fds-spacing-1);
  grid-template-areas:
    'inputbox toggle'
    'id id';
  grid-template-columns: 1fr auto;
}

.inputbox {
  grid-area: inputbox;
}

.id {
  align-items: center;
  display: flex;
  gap: var(--fds-spacing-1);
  grid-area: id;
}

.id code {
  align-items: center;
  align-self: stretch;
  display: flex;
}

.toggle {
  grid-area: toggle;
  border-radius: var(--fds-border_radius-medium);
  height: fit-content;
}

.toggle button {
  min-height: 2.1rem;
  min-width: 2.1rem;
}

.toggle [role='radiogroup'] {
  box-sizing: border-box;
}

.icon {
  font-size: var(--fds-sizing-5);
}

```